### PR TITLE
Scm docs

### PR DESCRIPTION
**Pull request by soltysh**
@bparees @danmcp @adellape ptal

### CODE REVIEW

File: architecture/additional_concepts/scm.adoc [hunk 1]
```diff
@@ -10,4 +10,15 @@
...
+OpenShift takes advantage of preexisting source control management (SCM) systems
+either hosted internally (in-house git server) or externally (eg. https://github.com/[github],
+https://bitbucket.org/[bitbucket], etc.), although initially supporting only
+https://git-scm.com/[GIT] solutions.
+
+SCM integration is tightly coupled with link:../core_concepts/builds_and_image_streams.html[builds],
+the two points being:
+
+- Creating a BuildConfig using a repository, which allows building your application
+inside of OpenShift. You can create such `*BuildConfig*` link:../dev_guide/builds.html#defining-a-buildconfig[manually]
```
Comment (adellape) [inline architecture/additional_concepts/scm.adoc]: Needs one more `../`

**Comment by adellape** [comment]
@soltysh One issue, LGTM, thanks!

**Comment by soltysh** [comment]
@adellape done.

**Comment by soltysh** [comment]
Thank @danmcp he pointed me we're missing here some info.

**Comment by bparees** [comment]
I think we'll want to flesh this out more at some point, but it's a good start, thanks @soltysh

File: architecture/additional_concepts/scm.adoc [hunk 2]
```diff
@@ -10,4 +10,15 @@
 toc::[]
 
 == Overview
-OpenShift source code repositories are referenced or owned by a build config and associated with projects.
\ No newline at end of file
+OpenShift takes advantage of preexisting source control management (SCM) systems
+either hosted internally (in-house git server) or externally (eg. https://github.com/[github],
+https://bitbucket.org/[bitbucket], etc.), although initially supporting only
+https://git-scm.com/[GIT] solutions.
```
Comment (rhcarvalho) [inline architecture/additional_concepts/scm.adoc]: minor suggestion to use the proper capitalization of those words/brands: GitHub, Bitbucket and Git.

**Comment by soltysh** [comment]
@rhcarvalho good catch, thx

**Pull request merged**